react-app/src/components/Header/index.js

```diff
@@ -1,40 +1,48 @@
-import React, { PureComponent } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 
 import Logo from 'components/Logo';
 import styles from './index.module.scss';
 
-export default class Header extends PureComponent {
-  static propTypes = {
-    /**
-     * The element (an ActionIcon) that this element will show in the left.
-     */
-    left: PropTypes.element,
-
-    /**
-     * The element (an ActionIcon) that this element will show in the right.
-     */
-    right: PropTypes.element,
-  };
-
-  static defaultProps = {
-    left: null,
-    right: null,
-  }
-
-  render() {
-    return (
-      <div className={styles.container}>
-        <div className={styles.left}>
-          {this.props.left}
-        </div>
-        <div className={styles.center}>
-          <Logo size={Logo.SMALL} />
-        </div>
-        <div className={styles.right}>
-          {this.props.right}
-        </div>
+const Header = ({ left, right, whiteLogo }) => {
+  const logo = Logo({ white: whiteLogo });
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.left}>
+        {left}
+      </div>
+      <div className={styles.center}>
+        {logo}
```

Comment:
`logo` is not some dynamic content or really long to justify extracting it into a variable. I think it makes more sense to just use it here like so:

```jsx
<div className={styles.center}>
  <Logo white={whiteLogo} />
</div>
```